tests/QueryTest.php

```diff
@@ -16,6 +16,7 @@ public function setUp(): void
         User::create(['name' => 'Brett Boe', 'age' => 35, 'title' => 'user']);
         User::create(['name' => 'Tommy Toe', 'age' => 33, 'title' => 'user']);
         User::create(['name' => 'Yvonne Yoe', 'age' => 35, 'title' => 'admin']);
+        User::create(['name' => 'Hazem Nassr', 'user-age' => 28, 'title' => 'member']);
```

Comment:
```suggestion
        User::create(['name' => 'John Smith', 'user-age' => 28, 'title' => 'member']);
```

I'm not happy with the fact you've added your name here, change it to something neutral.